acme/acme/crypto_util.py

```diff
@@ -203,7 +203,7 @@ def gen_ss_cert(key, domains, not_before=None,
     """
     assert domains, "Must provide one or more hostnames for the cert."
     cert = OpenSSL.crypto.X509()
-    cert.set_serial_number(1337)
+    cert.set_serial_number(int.from_bytes(OpenSSL.rand.bytes(16),'big'))
```

Comment:
I'm learning slowly - thank you for your patience.

Currently leaning towards:

`
    try:
        cert.set_serial_number(int(OpenSSL.rand.bytes(16).encode("hex"), 16))
    except AttributeError:
        cert.set_serial_number(int.from_bytes(OpenSSL.rand.bytes(16),'big'))
`

Does that look reasonable?